### DISCUSSION TITLE
The Sports DB

### DISCUSSION THREAD

**Discussion by wayner9** [OpenSageTV/Community#152] [General]
This is somewhat peripheral to SageTV, but I thought that I would post this in case anyone finds it useful.

I recently found out about https://www.thesportsdb.com/ which is a sports databases similar to TVDB or TMDB - it has metadata and fanart for sporting events.  It would have been nice to have this integrated into SageTV or BMT for SageTV, but that is unlikely to happen.  But there are Plugins for SageTV alternatives like Plex and Kodi available, see the forums here https://tsdb.club/Forum-Presentation-of-your-projects.

**Comment by emveepee** [comment]
I have been watching that for a while. I mainly watch sports in near-live or the next day and then delete and don't care about metadata but it could be a good source for SRE.  I run a NextPVR variant of the SRE service for NextPVR users and I would only need to pay $7 month for everyone should the ESPN feed fail.
